GHRSST_tutorial/basics_of_python/notebooks/beginner/notebooks/modules_and_packages.ipynb

```diff
@@ -126,7 +126,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.5.4"
```

Comment:
we have different versions of python. i have 3.7.1.

Reply:
i think i have 3.7.1 too, but hopefullly this won't matter.